blocks/hero-slider/hero-slider.js

```diff
@@ -0,0 +1,95 @@
+import { buildBlock, decorateBlock, loadBlock } from '../../scripts/aem.js';
+
+const MAX_BANNERS = 5;
+const AUTO_SCROLL_COUNT = 3;
```

Comment:
Shouldn't this be coming from documents as a config ? or these are the default one ?

Reply:
changed the default values as follows to avoid adding these values to the document
MAX_BANNERS : 15 to keep a safety check on the total number of assets added to the doc
AUTO_SCROLL_COUNT = -1 for all items to be scrolled by default